### PR TITLE
Update Tomba Dougoula > Tomba Kanssa (geom, capital_of, inception)

### PR DESCRIPTION
**Pull request by 5212BABA**
@stepps00 @nvkelso il faut mes modifications si c est  valable

### CODE REVIEW

File: data/126/021/452/3/1260214523.geojson [hunk 1]
```diff
@@ -6,21 +6,21 @@
     "edtf:inception":"uuuu",
     "geom:area":0.0,
     "geom:area_square_m":0.0,
-    "geom:bbox":"-10.2,11.73333,-10.2,11.73333",
+    "geom:bbox":"-10.18,11.73333,-10.18,11.73333",
```
Comment (nvkelso) [inline data/126/021/452/3/1260214523.geojson:9]: NOTE: These `geom` properties are auto-generated. To change them you'd need to edit the `geometry` section below (which it looks like you did, good). Generally don't manually edit these...

File: data/126/021/452/3/1260214523.geojson [hunk 2]
```diff
@@ -6,21 +6,21 @@
...
     "geom:latitude":11.73333,
-    "geom:longitude":-10.2,
+    "geom:longitude":-10.18,
     "gn:admin1_code":"K",
     "gn:admin2_code":"07",
-    "gn:asciiname":"Tomba Dougoula",
+    "gn:asciiname":"Tomba kanssa",
```
Comment (nvkelso) [inline data/126/021/452/3/1260214523.geojson:14]: GeoNames (`gn` prefixed properties) shouldn't be modified, they preserve what GeoNames knew about the place when we imported it.

File: data/126/021/452/3/1260214523.geojson [hunk 3]
```diff
@@ -6,21 +6,21 @@
...
     "gn:country_code":"GN",
     "gn:dem":403,
-    "gn:feature_class":"P",
-    "gn:feature_code":"PPL",
+    "gn:feature_class":"A",
```
Comment (nvkelso) [inline data/126/021/452/3/1260214523.geojson:17]: WOF uses "capital_of" properties to record these kind of things instead.

See also:

- https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/capital_of.json
- Example: https://spelunker.whosonfirst.org/id/85922845/

File: data/126/021/452/3/1260214523.geojson [hunk 4]
```diff
@@ -6,21 +6,21 @@
...
+    "gn:feature_code":"ADM3",
     "gn:geonameid":2414641,
     "gn:latitude":11.73333,
     "gn:longitude":-10.2,
-    "gn:modification_date":"2013-08-06",
-    "gn:name":"Tomba Dougoula",
+    "gn:modification_date":"2023-12-07",
```
Comment (nvkelso) [inline data/126/021/452/3/1260214523.geojson:22]: WOF exportify tool will generate a date in unix timestamp format for you automatically, but puts it in `wof:modified`.

File: data/126/021/452/3/1260214523.geojson [hunk 5]
```diff
@@ -54,7 +54,7 @@
     ],
     "wof:id":1260214523,
     "wof:lastmodified":1566639591,
-    "wof:name":"Tomba Dougoula",
+    "wof:name":"Tomba kanssa",
```
Comment (nvkelso) [inline data/126/021/452/3/1260214523.geojson:57]: Nit: capital K in `Tomba kanssa`.

**Review by nvkelso — CHANGES_REQUESTED** [review]
Since I merged my PR for this place in https://github.com/whosonfirst-data/whosonfirst-data-admin-gn/pull/41...

The only bit not captured that you could PR again (start over please versus updating this PR) is the changes in the geometry coordinate (and auto generated bbox and geom properties), if you want to specify the admin capital status, and the inception date (1915?).

**Comment by stepps00** [comment]
I believe the record in this PR was updated separately to resolve some of these issues. @nvkelso @5212BABA are we OK to close this PR/branch?

**Comment by nvkelso** [comment]
Yes, this PR can be closed.

If we wanted to do any follow ups, they are listed in https://github.com/whosonfirst-data/whosonfirst-data-admin-gn/pull/39#pullrequestreview-1770872205 but as rainy day.

**Pull request closed without merging**